database/migrations/2024_10_24_131715_fix_mpls_sdp_bindings_enum_typo.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::table('mpls_sdp_binds', function (Blueprint $table) {
```

Comment:
As far as I know you can simplify this by just doing:

```
        Schema::table('mpls_sdp_binds', function (Blueprint $table) {
            $table->string('sdpBindVcType', 30)->nullable()->change();
        });
```

You can't go back the other though as you've found and worked around but this will simplify the migration.

We do have the same issue with `sdpBindRowStatus`, `sdpBindAdminStatus`, `sdpBindOperStatus` and `sdpBindType`. Would you mind including these in that change please?

Reply:
Done.